tools/cellranger.wdl

```diff
@@ -81,24 +85,21 @@ task count {
         File raw_matrix = glob("*/outs/raw_feature_bc_matrix/matrix.mtx.gz")[0]
         File mol_info_h5 = glob("*/outs/molecule_info.h5")[0]
         File web_summary = glob("*/outs/web_summary.html")[0]
-        File loupe = glob("*/outs/cloupe.cloupe" )[0]
+        File loupe = glob("*/outs/cloupe.cloupe" )[0]  # TODO should this be "cloupe" instead of "loupe"?
```

Comment:
I have no idea what this file is. But looks to me like there's a missing `c` in the WDL